README.rst

```diff
@@ -909,13 +909,14 @@ or are using their free open source plan.
 Exit codes
 ----------
 
-Exit code are build using following bit flags:
+Exit code is build using following bit flags:
 
 *  0 OK
 *  4 Some files have invalid syntax (only in dry-run mode)
 *  8 Some files need fixing (only in dry-run mode)
 * 16 Configuration error of the application
 * 32 Configuration error of a Fixer
+* 64 Exception raised within the application
```

Comment:
I like the idea.

Please update https://github.com/bankiru/PHP-CS-Fixer/blob/master/UPGRADE.md#exit-codes as well